learnings.md

```diff
@@ -0,0 +1,15 @@
+# X11 forwarding
+To forward RVIZ and other windows from the Ubuntu Master to the Host Machine.
+If everything is set up correctly you have to do the following steps:
+1. On the Ubuntu Master, run`export DISPLAY=[Host Machine]:0`
```

Comment:
There is a space missing between "run" and the command

Reply:
fixed